doc/src/manual/methods.md

```diff
@@ -533,6 +533,63 @@ but are often preferred for clarity.
 Multiple parameters can be separated with commas, e.g. `where {T, S<:Real}`, or written using
 nested `where`, e.g. `where S<:Real where T`.
 
+### Methods with anonymous arguments
+
+Methods can be defined with anonymous arguments, where the value of the argument is not bound to a variable.
+This pattern can be helpful for indicating that the behavior of the function with respect to that argument only depends on the type of the input and not its value.
```

Comment:
Maybe wrap this line (I think generally lines should have < 100 chars, though I am not sure if this is a written rule; but it is certainly something the rest of this file adheres to).

Reply:
Done, thanks!